utils/fetchProjects.js

```diff
@@ -9,18 +10,196 @@ const db = require("../models/sequelize");
 const Users = db.users;
 const Projects = db.projects;
 const Projects_Repositories = db.projects_repositories;
+const Repositories = db.repositories;
 const Users_projects = db.users_projects;
 
+//function for get project info by project repo url
+const getInfoByProjectUrl = (url) => {
+  let project = {};
+  const splitArray = url.split("/");
+  if (splitArray.length > 4) {
```

Comment:
will this expression work in all cases? .length>4 ? @sahilbhatia if it should be like this only, please resolve this comment.